README.md

```diff
@@ -61,25 +61,27 @@ You can also make params optional by adding a `?` to it.
 
 ### Lazy Loading
 
-Lazy loading (code splitting) with `preact-router` can be implemented easily using the [AsyncRoute](https://www.npmjs.com/package/preact-async-route) module:
-
+~~Lazy loading (code splitting) with `preact-router` can be implemented easily using the [AsyncRoute](https://www.npmjs.com/package/preact-async-route) module~~.
```

Comment:
Let's remove this section entirely. No need to keep old information lying around :+1:

Reply:
Maybe we should just shorten it to:

> Lazy loading (code splitting) with `preact-router` can be implemented using the `lazy`-component.
>
> [Snippet here]